index.html

```diff
@@ -249,6 +252,87 @@ <h2>Introduction</h2>
         </li>
       </ul>
 
+      <section class="informative">
+        <h3>Intended Audience</h3>
+
+        <p>
+The intended audience for the Verifiable Credentials Data Model 1.0 W3C Recommendation (Recommendation) 
+is a broad range of professionals interested in furthering
+the application of verifiable credentials technology for use in software apps, agents, and services.
```

Comment:
Hmm, I read the meaning of this as "the intended audience for the VC data model spec is people interested in using VC technology" -- which, to me, seems circular and -- therefore, doesn't really add anything for the reader.

I'd think the intended audience would be people that want to express physical or digital documents that include assertions about other people, places, or things in a standard digital format so that they can be more widely consumed -- and so that the documents' authorship can be verified by third parties (verifiers). Or something along those lines. However, this sort of text may already be gleaned from the spec -- so it may just be a restatement of that.

Reply:
> Hmm, I read the meaning of this as "the intended audience for the VC data model spec is people interested in using VC technology" -- which, to me, seems circular and -- therefore, doesn't really add anything for the reader.

@dlongley What aspect do you think is _circular_?

> I'd think the intended audience would be people that want to express physical or digital documents that include assertions about other people, places, or things in a standard digital format so that they can be more widely consumed -- and so that the documents' authorship can be verified by third parties (verifiers). Or something along those lines. However, this sort of text may already be gleaned from the spec -- so it may just be a restatement of that.

@dlongley I agree that your comments represent a deeper level of detail. I have no problem adding something like:

> More specifically, the primary audience is software architects and developers that want to express physical or digital documents that include assertions about other people, places, or things in a standard digital format so that they can be more widely consumed.

...or something like that.  I'll wait before making this change.

Reply:
@mwherman2000,

>> Hmm, I read the meaning of this as "the intended audience for the VC data model spec is people interested in using VC technology" -- which, to me, seems circular and -- therefore, doesn't really add anything for the reader.
>>
> @dlongley What aspect do you think is circular?

To me, it is saying the intended audience of a spec that defines X are people that are interested in X. This seems largely vacuous to me. It's like reusing the word in its own definition -- and why I described it as circular.

An alternative to this would be to talk about people that are interested in other things that are components of or similar to X -- and therefore a spec that defines X subsumes their interests and standardizes some important element for them to achieve interoperability.

As an example: The intended audience of the 'Fantastic Authorization Token' spec are people who are trying to build interoperable libraries that grant third party applications access to restricted resources.

As a counter example: The intended audience of the 'Fantastic Authorization Token' spec are people who are interested in using Fantastic Authorization Token technology.

Reply:
@dlongley first, thank you for the dialog and moving this forward

Second, counter example:
> This is a football book (or book about football) that is expected to be of interest to people interested in professional football ...a bit wordy

Reply:
> "the intended audience for the VC data model spec is people interested in using VC technology"

@dlongley IMOyour paraphrasing isn't quite right.  Try this:

> "the intended audience for [this document aka Recommendation] is people interested in using VC technology"

I think this makes sense (and the original wording more correct) ...but I will endeavor to refine it some more in the next go-around.

Reply:
If I were copy editing something that had this statement:

"This is a football book (or book about football) that is expected to be of interest to people interested in professional football ..."

I'd recommend cutting it because it doesn't add value. Of course a football book is expected to be of interest to people interested in football. If the statement is to be of value, it should explain a bit more, for example, this is a football book that is expected to be of interest to people who would like to know more about defensive strategies to employ against well known offensive strategies (or whatever).

I'd say that the goal of mentioning an intended audience for a document is to narrow (or highlight) something that wouldn't otherwise be known from the title of the document or the rest of its basic introduction. If we're not adding more information with the statement, then I'd consider it is a net negative on the document; it makes it more verbose or complex which is an unnecessary burden on readers and/or maintainers. I think we should want to make a specification as short as possible (but not shorter) :).